internal/machine/leasable_machine.go

```diff
@@ -536,13 +537,7 @@ func (lm *leasableMachine) ReleaseLease(ctx context.Context) error {
 		defer cancel()
 	}
 
-	err := lm.flapsClient.ReleaseLease(ctx, lm.machine.ID, nonce)
-	contextTimedOutOrCanceled := errors.Is(err, context.DeadlineExceeded) || errors.Is(err, context.Canceled)
-	if err != nil && (!contextWasAlreadyCanceled || !contextTimedOutOrCanceled) {
```

Comment:
does it make any difference for `contextWasAlreadyCanceled` and `!contextTimedOutOrCanceled` ?

Reply:
Good catch! There are places where we call ReleaseLease and not logging any errors from that. Let me adjust them.